_includes/integrations/API/partial_backend_and_storefront_api_differences.md

```diff
@@ -0,0 +1,131 @@
+This document describes differences between the backend and storefront code in API modules.
```

Comment:
Double-check if storefront is used correctly here. We use this to refer to our frontend solution. Here, you might use `frontend`